e2e/firstTest.spec.js

```diff
@@ -7,22 +7,24 @@ describe("Main screen", () => {
   });
 
   /**
-  * The following source code represents the following automated test scenario
-  * 1. Load the app
-  * 2. Display main screen
-  * 3. Click on the searchbar
-  * 4. Type the query: "hall building"
-  * 5. Select the corresponding item from the dropdown list
-  * 6. Click on the bottom menu (not yet implemented)
-  * 7. Scroll down and read the information provided (not yet implemented)
-  */
+   * The following source code represents the following automated test scenario
+   * 1. Load the app
+   * 2. Display main screen
+   * 3. Click on the searchbar
+   * 4. Type the query: "hall building"
+   * 5. Select the corresponding item from the dropdown list
+   * 6. Click on the bottom menu (not yet implemented)
+   * 7. Scroll down and read the information provided (not yet implemented)
```

Comment:
Not yet implemented because the feature doesn't exist or because we couldn't get this part of the test running?